src/tasks/lm.py

```diff
@@ -152,4 +152,61 @@ def __init__(self, path, *args, **kw):
                                'val': os.path.join(path, "valid.sentences.txt"),
                                'test': os.path.join(path, "test.sentences.txt")}
 
+@register_task('wsj', rel_path='Penn/')
+class WSJLanguageModelling(LanguageModelingTask):
+    """ Language modeling on a PTB dataset
+    See base class: LanguageModelingTask
+    """
+    def count_examples(self):
+        """Computes number of samples
+        Assuming every line is one example.
+        """
+        example_counts = {}
+        for split, split_path in self.files_by_split.items():
+            arr = [line.strip().split()+["<EOS>"] for line in open(split_path)]
+            allf = 0
+            for x in arr:
+                allf += len(x)
+            example_counts[split] = int(math.ceil(allf/self.max_seq_len))
+        self.example_counts = example_counts
+
```

Comment:
Fix style.

Reply:
I'm assuming the point here was to add spaces b/w the int(_). I ask because PEP didn't throw up an error.